src/Symfony/Component/Form/Tests/AbstractLayoutTest.php

```diff
@@ -2668,6 +2669,42 @@ public function testHelpWithTranslationParameters()
         );
     }
 
+    /**
+     * @requires function Symfony\Component\Translation\TranslatableMessage::__construct
+     */
+    public function testLabelWithTranslatableMessage()
+    {
+        $form = $this->factory->createNamed('name', 'Symfony\Component\Form\Extension\Core\Type\TextType', null, [
+            'label' => new TranslatableMessage('foo'),
+        ]);
+        $html = $this->renderLabel($form->createView());
+
+        $this->assertMatchesXpath($html,
+            '/label
+    [@for="name"]
+    [.="[trans]foo[/trans]"]
+'
+        );
+    }
+
+    /**
+     * @requires function Symfony\Component\Translation\TranslatableMessage::__construct
```

Comment:
This doesn't look needed.

Reply:
indeed. found it elsewhere but will remove it.